keyboards/xbows/woody/readme.md

```diff
@@ -0,0 +1,15 @@
+# WOO-DY
+![WOO-DY](https://ftp.bmp.ovh/imgs/2019/10/cf5ba2fa02a62afc.png)
+
+Woo-dy, A 67 Keys Wooden Mechanical Keyboard.
+At once minimalist and stylish, Woo-dy brings a natural calm and relaxation to your working space. The natural wood base made from real walnut or cherry reduces weight, feels great and adds subtle, natural style to any room.
+
+Keyboard Maintainer: X-BOWS  
```

Comment:
@moyi4681 are you the maintainer for these boards? If so, you should put your name here.

Reply:
we agreed to make  maintainer is xbows